src/tests/test_topo.C

```diff
@@ -163,10 +147,37 @@ TEST(Topo, SpinupContinuation)
 	EXPECT_EQ(failed, false);
 }
 
+//------------------------------------------------------------------
+TEST(Topo, RHS)
+{
+    bool failed = false;
+    try
+    {
+        topo->setPar("Delta", 0.3);
+        topo->computeRHS();
+        topo->computeJacobian();
+        topo->preProcess();
+    }
+    catch (...)
+    {
+        failed = true;
+        throw;
+    }
+    EXPECT_EQ(failed, false);
+}
+
 //------------------------------------------------------------------
 TEST(Topo, TopoContinuation)
 {
 	bool failed = false;
+
+    // create state diff vector
+    Ocean::VectorPtr stateDiff = ocean->getState('C');
```

Comment:
Why is this called stateDiff?

Reply:
fixed in f18d0733903d8a2dd14d29db931cbd2625f1238f